dfpl/options.py

```diff
@@ -1034,141 +1226,151 @@ def parseTrainGnn(parser: argparse.ArgumentParser) -> None:
     )
 
 
-def parseInputPredict(parser: argparse.ArgumentParser) -> None:
-    """
-    Parse the input arguments.
-
-    :return: A namespace object built up from attributes parsed out of the cmd line.
-    """
-
+def parsePredictGnn(parser: argparse.ArgumentParser) -> None:
     general_args = parser.add_argument_group("General Configuration")
     files_args = parser.add_argument_group("Files")
+    uncertainty_args = parser.add_argument_group("Uncertainty Configuration")
+
+    general_args.add_argument(
+        "--checkpoint_path",
+        type=str,
+        metavar="FILE",
+        help="Path to model checkpoint (.pt file)",
+    )
+    # general_args.add_argument(
+    #     "--no_features_scaling",
+    #     action="store_true",
+    #     help="Turn on scaling of features",
+    # )
     files_args.add_argument(
         "-f",
         "--configFile",
-        metavar="FILE",
         type=str,
-        help="Input JSON file that contains all information for training/predicting.",
-        default=argparse.SUPPRESS,
-    )
-    files_args.add_argument(
-        "-i",
-        "--inputFile",
         metavar="FILE",
-        type=str,
-        help="The file containing the data for the prediction in (unquoted) "
-        "comma separated CSV format. The column named 'smiles' or 'fp'"
-        "contains the field to be predicted. Please adjust the type "
-        "that should be predicted (fp or smile) with -t option appropriately."
-        "An optional column 'id' is used to assign the outcomes to the"
-        "original identifiers. If this column is missing, the results are"
-        "numbered in the order of their appearance in the input file."
-        "A header is expected and respective column names are used.",
-        default=argparse.SUPPRESS,
+        help="Path to a .json file containing arguments. Any arguments present in the config"
+        "file will override arguments specified via the command line or by the defaults.",
     )
     files_args.add_argument(
-        "-o",
-        "--outputDir",
-        metavar="DIR",
+        "--test_path",
         type=str,
-        help="Prefix of output directory. It will contain a log file and the file specified"
-        "with --outputFile.",
-        default=argparse.SUPPRESS,
+        help="Path to CSV file containing testing data for which predictions will be made.",
```

Comment:
```suggestion
        help="Test data file. CSV file containing test data for which predictions will be made.",
```

Reply:
Does CSV really mean comma separated? Or does TSV also work?

Reply:
I haven't tried tsv, but I am sure csv does (and yes it means comma separated).

Reply:
Thanks for the clarification.